djoser/urls.py

```diff
@@ -0,0 +1,90 @@
+from django.conf.urls import include, url
+from django.contrib.auth import get_user_model
+
+from rest_framework.routers import DefaultRouter, DynamicListRoute, Route
+
+from djoser import views
+
+User = get_user_model()
+
+
+class DjoserRouter(DefaultRouter):
+    def __init__(self, include_token_urls=False):
+        super(DjoserRouter, self).__init__()
+        self.trailing_slash = '/?'
+
+        self._insert_routes()
+        self._register_urls(include_token_urls)
+
+    def _insert_routes(self):
+        self.routes.insert(0, Route(
+            url=r'^{prefix}{trailing_slash}$',
+            mapping={
+                'delete': 'remove_user',
+                'get': 'me',
+                'put': 'update',
+            },
+            name='{basename}-instance',
+            initkwargs={'suffix': 'Instance'}
+        ))
+        self.routes.insert(0, Route(
+            url=r'^{prefix}{trailing_slash}$',
```

Comment:
I think this conflicts with **very** similar Route in [line 20](https://github.com/sunscrapers/djoser/pull/259/files#diff-2700b7139be91e23d6319ef703a291b7R20).